packages/react-charts/src/components/ChartLegendTooltip/ChartLegendTooltip.tsx

```diff
@@ -49,7 +49,7 @@ export interface ChartLegendTooltipProps extends ChartCursorTooltipProps {
   /**
    * The angle prop specifies the angle to rotate the tooltip around its origin point.
    */
-  angle?: string | number;
+  angle?: number;
```

Comment:
Would this break people who are passing a string?

Reply:
This must match the current Victory type in order to compile. I suspect Victory corrected the type because only a number was supported.